llvm/test/Instrumentation/AddressSanitizer/do-not-instrument-globals-windows.ll

```diff
@@ -0,0 +1,10 @@
+; This test checks that we are not instrumenting unnecessary globals
```

Comment:
I feel that we should better organize global variable tests by combining some into one file. They are currently scattered everywhere.

basic-msvc64.ll may be a better file for this test since basic.ll tests ELF available_externally.